### PR TITLE
Catch errors when we fail to modify targetgroup targets

### PR DESCRIPTION
**Pull request by bigkraig**

### CODE REVIEW

File: controller/alb/targetgroup.go [hunk 1]
```diff
@@ -138,8 +138,8 @@ func (tg *TargetGroup) create(lb *LoadBalancer) error {
 		Port:                       tg.DesiredTargetGroup.Port,
 		Protocol:                   tg.DesiredTargetGroup.Protocol,
 		Name:                       tg.DesiredTargetGroup.TargetGroupName,
-		UnhealthyThresholdCount:    tg.DesiredTargetGroup.UnhealthyThresholdCount,
-		VpcId:                      lb.CurrentLoadBalancer.VpcId,
+		UnhealthyThresholdCount: tg.DesiredTargetGroup.UnhealthyThresholdCount,
+		VpcId: lb.CurrentLoadBalancer.VpcId,
```
Comment (joshrosso) [inline controller/alb/targetgroup.go]: Does this need `go fmt`?

File: controller/ingress.go [hunk 2]
```diff
@@ -234,8 +234,8 @@ func assembleIngresses(ac *ALBController) ALBIngressesT {
 		ingressID := namespace + "-" + ingressName
 
 		rs := &alb.ResourceRecordSet{
-			IngressID:                &ingressID,
-			ZoneID:                   zone.Id,
+			IngressID: &ingressID,
+			ZoneID:    zone.Id,
```
Comment (joshrosso) [inline controller/ingress.go]: Does this need `go fmt`?

**Comment by bigkraig** [comment]
i think my gofmt has lost its mind

**Comment by joshrosso** [comment]
Nope, actually I think my go fmt has lost its mind. I was looking over those things as if they were struct definitions, but that spacing isn't required for composite literal definitions.

Don't really know how the spacing got that way in the first place.

**Pull request merged**